scripts/install-kubefed.sh

```diff
@@ -87,10 +86,8 @@ elif test X"$LOCATION" = Xolm-kube; then
  ./scripts/kubernetes/olm-install.sh ${OLM_VERSION}
  
  echo "OLM is deployed on kube cluster"
- cp $CSV_PATH $CSV_TMP_PATH
- chmod +w $CSV_TMP_PATH
- sed "s,image: quay.*$,image: ${IMAGE_NAME}," -i $CSV_TMP_PATH|./hack/catalog.sh | kubectl apply -n $NAMESPACE -f -
- rm $CSV_TMP_PATH
+ sed "s,image: quay.*$,image: ${IMAGE_NAME}," -i.bak $CSV_PATH|./hack/catalog.sh | kubectl apply -n $NAMESPACE -f -
```

Comment:
Can you explain this change? Why is it needed?

Reply:
In the beginning, we changed the kubefed-operator image(`$IMAGE_NAME`) in the `$CSV_TMP_PATH`, but in the `./hack/catalog.sh`, we did not use the `$CSV_TMP_PATH`, and still used the `$CSV_PATH`. So, here, I updated the `$CSV_PATH` directly, so can make the changing can be used by `./hack/catalog.sh`.

Reply:
@abhat I created a new commit to avoid updating csv file directly.